src/components/verification-grid/verification-grid.ts

```diff
@@ -288,15 +296,17 @@ export class VerificationGridComponent extends WithShoelace(AbstractComponent(Li
   public static readonly decisionMadeEventName = "decision-made";
   private static readonly loadedEventName = "grid-loaded";
   private static readonly autoPageTimeout = 0.3 satisfies Seconds;
+  private static readonly defaultGridTileTemplateId = "default-tile-template";
 
   @provide({ context: verificationGridContext })
-  @state()
+  @property({ attribute: false })
   public settings: VerificationGridSettings = {
-    showAxes: signal(true),
-    showMediaControls: signal(true),
     isFullscreen: signal(false),
   };
 
+  @provide({ context: spectrogramOptionsContext })
+  public spectrogramOptions = signal<Partial<SpectrogramOptions>>({});
```

Comment:
Why is this a signal, and why is it public?

Reply:
It doesn't need to be. Change to `private` (really `protected` to stop tree shaking)